config-model/src/main/java/com/yahoo/vespa/model/content/ContentSearchCluster.java

```diff
@@ -336,6 +336,28 @@ public void handleRedundancy(Redundancy redundancy) {
                 .collect(Collectors.toList());
     }
 
+    public List<NewDocumentType> getDocumentTypesWithStreamingCluster() {
```

Comment:
I don't think we need this, as we should be happy checking whether a document changes between "index" and not "index".

Reply:
It's the transitions between indexing modes we want to react on. We have to check if the document type is new or removed otherwise.

Reply:
But a document type must be either indexed or not — it would suffice to check whether it went from indexed to not indexed, or the other way, I believe.